cmd/podman/containers/ps.go

```diff
@@ -199,12 +200,6 @@ func ps(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	if len(listOpts.Sort) > 0 {
```

Comment:
Doesn't that break `podman ps --sort`?

Reply:
No, this code is repeated from https://github.com/containers/podman/pull/17627/files#diff-d1aefcf956b4e079a0684a5f5a5d545d4be20870f172fdac95c77cac1fd90482R179. I noticed it while updating ps but can remove it if it is preferred to do in another PR.

Reply:
Breaking it out into another PR or commit would be great, thanks!

Reply:
Removed and will open another PR to fix.